### PR TITLE
i#4550: Support EVEX encoded vsqrt instructions.

### LINKED ISSUE #4550: EVEX encoded vsqrtps/vsqrtpd are missing
```
khuey@minbar:~/dev/dynamorio$ ./obj/clients/bin64/drdisas -syntax intel c5 fc 51 ca
 c5 fc 51 ca          vsqrtps ymm1, ymm2
khuey@minbar:~/dev/dynamorio$ ./obj/clients/bin64/drdisas -syntax intel 62 f1 7c 29 51 ca
 62 f1 7c 29 51 ca...??  <INVALID>
disassembly failed: invalid instruction
khuey@minbar:~/dev/dynamorio$ ./obj/clients/bin64/drdisas -syntax intel c5 fd 51 ca
 c5 fd 51 ca          vsqrtpd ymm1, ymm2
khuey@minbar:~/dev/dynamorio$ ./obj/clients/bin64/drdisas -syntax intel 62 f1 fd 29 51 ca
 62 f1 fd 29 51 ca...??  <INVALID>
disassembly failed: invalid instruction
```

### PR DESCRIPTION
**Pull request by khuey**
Adds support for the EVEX encodings of vsqrtps, vsqrtpd, vsqrtss, and vsqrtsd. Also adds API tests and uncomments the relevant sections of the binutils decoder tests, with updated expectation files.

Fixes #4550

### CODE REVIEW

**Comment by khuey** [comment]
I don't think that test failure is real.

**Comment by derekbruening** [comment]
> I don't think that test failure is real.

The convention is to list the test name with a link to the filed issue (or to file a new one if convinced it's a flake).

File: core/ir/x86/decode_table.c [hunk 1]
```diff
@@ -9206,6 +9214,16 @@ const instr_info_t evex_Wb_extensions[][4] = {
     {OP_vcvtps2ph, 0x663a1d18, "vcvtps2ph", Uqq, xx, KEw, Voq, Ib, mrm|evex|sae|reqp|tthvm, x, END_LIST},
     {INVALID, 0, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
     {INVALID, 0, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+  }, { /* evex_W_ext 265 */
+    {OP_vsqrtps, 0x0f5100, "vsqrtps", Ves, xx, KEw, Wes, xx, mrm|evex|ttfv, x, modx[118][0]},
+    {MOD_EXT, 0x0f5110, "(mod ext 118)", xx, xx, xx, xx, xx, mrm|evex, x, 118},
+    {OP_vsqrtpd, 0x660f5140, "vsqrtpd", Ved, xx, KEb, Wed, xx, mrm|evex|ttfv, x, modx[119][0]},
+    {MOD_EXT, 0x660f5150, "(mod ext 119)", xx, xx, xx, xx, xx, mrm|evex, x, 119},
+  }, { /* evex_W_ext 266 */
+    {OP_vsqrtss, 0xf30f5100, "vsqrtss", Vdq, xx, KE1b, Hdq, Wss, mrm|evex|ttt1s, x, tevexwb[266][1]},
```
Comment (derekbruening) [inline core/ir/x86/decode_table.c:9223]: (OK, tuple types of Full Vector and Typle 1 Scalar seem to match.)

**Review by derekbruening — APPROVED** [review]
Thank you for fixing this!

**Comment by khuey** [comment]
> 
> 
> > I don't think that test failure is real.
> 
> The convention is to list the test name with a link to the filed issue (or to file a new one if convinced it's a flake).

I had trouble finding a useful error message in the log, other than just that decode-stress failed.

**Comment by derekbruening** [comment]
> > > I don't think that test failure is real.
> > 
> > 
> > The convention is to list the test name with a link to the filed issue (or to file a new one if convinced it's a flake).
> 
> I had trouble finding a useful error message in the log, other than just that decode-stress failed.

It seems to be there.  The output is both above as verbose original and below as why it didn't match:
```
<Application /home/runner/work/dynamorio/dynamorio/build_debug-internal-32/suite/tests/bin/common.decode (19098).  Internal Error: DynamoRIO debug check failure: /home/runner/work/dynamorio/dynamorio/core/translate.c:1094 false
```
That is #4723.

**Comment by khuey** [comment]
> 
> 
> > > > I don't think that test failure is real.
> > > 
> > > 
> > > The convention is to list the test name with a link to the filed issue (or to file a new one if convinced it's a flake).
> > 
> > 
> > I had trouble finding a useful error message in the log, other than just that decode-stress failed.
> 
> It seems to be there. The output is both above as verbose original and below as why it didn't match:
> 
> ```
> <Application /home/runner/work/dynamorio/dynamorio/build_debug-internal-32/suite/tests/bin/common.decode (19098).  Internal Error: DynamoRIO debug check failure: /home/runner/work/dynamorio/dynamorio/core/translate.c:1094 false
> ```
> 
> That is #4723.

Ah, great.

**Pull request merged**